sdk/core/core/test/curl_easy_perform_poc.c

```diff
@@ -106,5 +108,8 @@ int main() {
   az_mut_span_set(temp_buf, 0);
   az_span_free(&temp_buf);
 
+  az_result result;
+  result = az_http_pipeline_process(&hrb, &http_buf_response);
```

Comment:
reuse `ignore_result`

Reply:
Leaving for now, in a subsequent PR this poc needs to switch from az_http_client_send_request to az_http_pipeline_process